src/panels/developer-tools/state/developer-tools-state.js

```diff
@@ -169,6 +173,12 @@ class HaPanelDevState extends EventsMixin(LocalizeMixin(PolymerElement)) {
                 title="[[localize('ui.panel.developer-tools.tabs.states.more_info')]]"
                 path="[[informationOutlineIcon()]]"
               ></ha-svg-icon>
+              <ha-svg-icon
+                on-click="copyEntity"
+                alt="[[localize('ui.panel.developer-tools.tabs.states.copy_id')]]"
+                title="[[localize('ui.panel.developer-tools.tabs.states.copy_id')]]"
+                path="[[clipboardOutlineIcon()]]"
```

Comment:
I think the icon can be put as a property. Like here: https://github.com/home-assistant/frontend/blob/c22022856616c10efa986dfc6cfe97854c1195e1/src/dialogs/more-info/controls/more-info-weather.ts#L86

```suggestion
                .path=${mdiClipboardTextMultipleOutline}
```

and `clipboardOutlineIcon()` can be removed

Reply:
Sorry, I just noticed this is still using Polymer. NVM.

Reply:
Getting on the build. Do you think that I should do that for the other icon, or keep the PR small?

Reply:
@KTibow sorry for the trouble, leave this as it. I noticed that this is in Polymer, so my initial idea won't work 😕

Reply:
Okay.